src/sage/rings/function_field/drinfeld_modules/homset.py

```diff
@@ -20,13 +20,85 @@
 #                   http://www.gnu.org/licenses/
 # *****************************************************************************
 
+import operator
+
 from sage.categories.drinfeld_modules import DrinfeldModules
 from sage.categories.homset import Homset
+from sage.categories.action import Action
 from sage.misc.latex import latex
 from sage.rings.function_field.drinfeld_modules.morphism import DrinfeldModuleMorphism
 from sage.structure.parent import Parent
 
 
+class DrinfeldModuleMorphismAction(Action):
+    r"""
+    Action of the function ring on the homset of a Drinfeld module.
+    """
+    def __init__(self, A, H, is_left, op):
+        r"""
+        Initialize this action.
```

Comment:
I believe we usually say *the* instead of *this*. See for example the documentation in `DrinfeldModule`, where we used *the Drinfeld module* a lot.

```suggestion
        Initialize the action.
```

Reply:
Not sure. Usually we use `this` to refer to `self` :-)

Reply:
Ok. We can leave it as is!